lib/eventwebhook/EventWebhook.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace SendGrid\EventWebhook;
+
+use EllipticCurve\Ecdsa;
+use EllipticCurve\PublicKey;
+use EllipticCurve\Signature;
+
+/**
+ * This class allows you to use the Event Webhook feature. Read the docs for
+ * more details: https://sendgrid.com/docs/for-developers/tracking-events/event
+ *
+ * @package SendGrid\EventWebhook
+ */
+class EventWebhook
+{
+    /**
+     * Convert the public key string to a ECPublicKey.
+     *
+     * @param string $publicKey verification key under Mail Settings
+     * @return PublicKey public key using the ECDSA algorithm
+     */
+    public function convertPublicKeyToECDSA($publicKey)
+    {
+        return PublicKey::fromString($publicKey);
+    }
+
+    /**
+     * Verify signed event webhook requests.
+     *
+     * @param PublicKey $publicKey elliptic curve public key
```

Comment:
Is this a `PublicKey` to have parity with Java & Go implementations? If we know users will be getting a base64 public key, why don't we abstract away the conversion by taking in the string and converting to ECDSA within `verifySignature`?

Reply:
I was trying to make the interface similar to the Java and Go implementations where it was a 2-step process. Perhaps the key will be re-used so no need to keep converting it.